### ISSUE TITLE
Kwargs in Execution of script/command with hyphen mark doesnt support in python

### ISSUE THREAD

**Issue by Narenthirasamy** [marklap/rundeckrun#24]
In rundeckrun/rundeck/api.py, the kwargs contains hyphen(-) mark, python doesn't allow this character in method and variable, so we cant use it api call. 
For reference: http://stackoverflow.com/questions/2064329/why-python-does-not-allow-hyphens

**Comment by marklap** [comment]
Well... that stinks. Lesson learned. I did a little experimentation and while it's not a solution, a workaround is to use a splat on a dictionary. It doesn't seem to mind that in python 2.7 nor python 3.4 (the two I tested).

    Python 2.7.6 (default, Mar 22 2014, 22:59:56) 
    [GCC 4.8.2] on linux2
    Type "help", "copyright", "credits" or "license" for more information.
    >>> def foo(**kw):
    ...   pass
    ... 
    >>> foo(this-is=1)
      File "<stdin>", line 1
        foo(this-is=1)
                ^
    SyntaxError: invalid syntax
    >>> foo(**{'this-is': 1})
    >>> 
    >>> exit()


    Python 3.4.0 (default, Apr 11 2014, 13:05:11) 
    [GCC 4.8.2] on linux
    Type "help", "copyright", "credits" or "license" for more information.
    >>> def foo(**kw):
    ...   pass
    ... 
    >>> foo(this-is=1)
      File "<stdin>", line 1
        foo(this-is=1)
                ^
    SyntaxError: invalid syntax
    >>> foo(**{'this-is': 1})
    >>> 
    >>> exit()

I liked the idea of keeping the kw args closely tied to the kw args that the Rundeck API expects so that's why I implemented them as such. Since this issue seems related to only host filters maybe if we expect them to be passed in as a dict that would suffice. Seems like the easiest solution to keep the keywords in parity with the API.

I'm open to suggestions.

**Comment by Narenthirasamy** [comment]
Yes as far as from client side, this approach should be fine, whereas we can raise this concern to rundeck server API team to fix this without hyphen.

**Comment by marklap** [comment]
I'll likely not be doing the work for this but will gladly review and accept pull requests.
